crates/ruff/src/rules/isort/helpers.rs

```diff
@@ -110,7 +110,7 @@ pub fn find_splice_location(body: &[Stmt], locator: &Locator) -> Location {
 
     // Find the first token that isn't a comment or whitespace.
     let contents = locator.skip(splice);
-    for (.., tok, end) in lexer::make_tokenizer(contents).flatten() {
+    for (.., tok, end) in lexer::make_tokenizer_located(contents, splice).flatten() {
```

Comment:
We're generating an offset here, so it needs to be relative to the current `splice`!